### ISSUE TITLE
Notes for QA

### ISSUE THREAD

**Issue by brettfiedler** [phetsims/photoelectric-effect#266]
- Orca (linux screen reader) reports percentage of slider/spinner. Do NVDA or VoiceOver do this? Specifically, wavelength slider, band depth slider, work function slider.
- Orca does not read greek phi symbol. Do VoiceOver and NVDA (@jessegreenberg reports yes for NVDA)?

**Comment by jessegreenberg** [comment]
> Do NVDA or VoiceOver do this?

No, our supported screen readers read teh aria-valuetext (or whatever the number is for the spinner/slider). @brettfiedler is there anything else you want to check for this one?
